### ISSUE TITLE
"How to" uploading models to GitHub

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/Suite#129]
During PAGE we have been approached by users who would like to share their models in the community but do not know how.

We need some kind of short "how to" description that will be linked on the website in the section "Models"

![image](https://github.com/Open-Systems-Pharmacology/Suite/assets/25903040/754fc488-05bf-4dfb-8a91-52f911be6b98)

Also, there is no link to the website on the github repo.

**Comment by PavelBal** [comment]
@Yuri05 @msevestre

**Comment by Yuri05** [comment]
https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CREATING_MODEL_REPOSITORY.md

**Comment by PavelBal** [comment]
Top! Bleibt also nur noch prominent platzieren ;)

**Comment by Yuri05** [comment]
exactly. "How to upload a model" button as in your screen would be good. 
We could also extend the OSP readme:
![grafik](https://github.com/Open-Systems-Pharmacology/Suite/assets/25061876/203c12ad-65e4-49fc-8d2a-2bd4e8fdd152)
